pages/src/source/dependencies/domumornamentum.md

```diff
@@ -27,6 +27,6 @@ When you have that information, you can put the materials in the Architects cutt
 <img src="../../assets/images/misc/do_shingles_cutter.png" alt="The Architects cutter's interface, showing shingles"/>
 
 When you have crafters, e.g. a [Sawmill](../../source/buildings/sawmill), up and running, you can then teach them the recipe by choosing the "custom recipe" tab and putting it in there.
-Input 1 is the top left slot of the cutter, input 2 is the top right, and input 3 the bottom left slot in the cutter. When you have put the items in the slots, you will see various items below the input slots. The crafter can create ALL of those items from the recipe you have input. Make sure you teach the recipe to the appropriate crafter, though!
+Input 1 is the top left slot of the cutter, input 2 is the top right, and input 3 the bottom left slot in the cutter. When you have put the items in the slots, you will see various items below the input slots. The crafter can create ALL of those items from the recipe you have input. Make sure you teach the recipe to the appropriate crafter, though! They can only learn a recipe if one of the materials used is appropriate for their profession.
```

Comment:
Rather than this, what if it were reworded to "corresponding crafter" instead of "appropriate crafter"?

Reply:
Why?  They seem close enough to synonymous to me.

Also I can’t easily find how to make a new commit to a PR on the GitHub online interface and downloading this to my computer would be a pain, but I’ll do it if I have to.

(There’s a *reason* I don’t edit this wiki as much as I would others; GitHub adds a lot of overhead.)

Reply:
I found the edit button for a PR, and I added more detail and an example, instead of just rewriting the one sentence.